### PR TITLE
06/06 added css

### PR DESCRIPTION
**Pull request by Jessica951215**
For the universal top bar, I separated the css files, using sass converted to css.

### CODE REVIEW

File: client/topbar.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import 'reflect-metadata';
+import 'zone.js/dist/zone';
+import { Component, provide } from '@angular/core';
+import { bootstrap } from 'angular2-meteor-auto-bootstrap';
```
Comment (DerekTBrown) [inline client/topbar.ts]: Need to remove Bootstrap.

File: public/sass/toolbar.sass [hunk 2]
```diff
@@ -0,0 +1,45 @@
+
```
Comment (DerekTBrown) [inline public/sass/toolbar.sass]: This file is missing lots of curly braces.

File: client/topbar.ts [hunk 3]
```diff
@@ -0,0 +1,37 @@
+import 'reflect-metadata';
+import 'zone.js/dist/zone';
```
Comment (DerekTBrown) [inline client/topbar.ts]: What is this for?

File: public/sass/toolbar.sass (hunk 2 above)
Comment (Jessica951215) [inline public/sass/toolbar.sass]: isn't indentation the syntax for sass? I thought no curly braces needed.

File: client/topbar.ts (hunk 3 above)
Comment (Jessica951215) [inline client/topbar.ts]: These two lines are included in every example of the angular2-meteor tutorial, I first thought it's just part of the project. However I tested and nothing changed when I took them out. I guess we can leave them out for now.

**Pull request closed without merging**